.husky/pre-push

```diff
@@ -21,7 +21,7 @@ fi
 
 # Run Unit Tests
 
-npm run test
+# npm run test
 
 # Run Linting
 
```

Comment:
Lets remove the test and lint checks entirely from the pre-push check. They don't need to be here.

Reply:
Uncommented